frontend/src/scenes/persons/PersonDisplay.tsx

```diff
@@ -23,6 +23,7 @@ export interface PersonDisplayProps {
     noEllipsis?: boolean
     noPopover?: boolean
     isCentered?: boolean
+    children?: React.ReactChild
```

Comment:
Silly question... ELI5

Why not import { ReactChild } above and then use it without the `React.` here?

Reply:
Tbh I have no valid reason to use one over the other 🤷 I guess the `React` global is available so I just start writing and it doesn't auto import

Quick scan of the codebase suggests that's how it's been done elsewhere
<img width="383" alt="Screenshot 2024-09-12 at 20 54 49" src="https://github.com/user-attachments/assets/a67d9a7c-2e2c-42a6-b748-27efd26eb3fc">